config/locales/en.yml

```diff
@@ -2952,6 +2957,14 @@ See the %{link} to find out more about %{sitename}'s features and to start using
     new_payment: "New Payment"
     capture: "Capture"
     void: "Void"
+    login: "Login"
+    password: "Password"
+    signature: "Signature"
+    solution: "Solution"
+    landing_page: "Landing Page"
+    server: "Server"
+    test_mode: "Test Mode"
+    logourl: "Logourl"
```

Comment:
there's already a group for payment methods here, can you move this to below line 3042? Thanks!

Reply:
I wonder where these translations are called from? If you know you could try to make them lazylookups and put them in their relative path. It's a stretch goal, maybe you want to do only in the next PR :+1: